### PR TITLE
FEATURE: Add keyboard shortcuts

### PR DESCRIPTION
**Pull request by pmusaraj**
Mostly for use in iPads/macOS. Use ⌘ + 1 through 9 to open sites 1-9. And ⌘ + W to dismiss an open site. 

Feature request: https://meta.discourse.org/t/navigating-between-discourse-fora-on-discoursehub/217523

### CODE REVIEW

File: js/Discourse.js [hunk 1]
```diff
@@ -310,6 +315,21 @@ class Discourse extends React.Component {
           }
         },
       );
+
+      eventEmitter.addListener('keyInputEvent', res => {
+        const {input} = res;
+        const index = parseInt(input - 1, 10);
```
[the remark this answers is not in the document]
Comment (pmusaraj) [reply js/Discourse.js]: Hmm, it was worked, but indeed it doesn't make sense. Fixed.

File: ios/Discourse.xcodeproj/xcshareddata/xcschemes/Discourse.xcscheme [hunk 2]
```diff
@@ -42,8 +42,8 @@
    </TestAction>
    <LaunchAction
       buildConfiguration = "Debug"
-      selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
-      selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      selectedDebuggerIdentifier = ""
+      selectedLauncherIdentifier = "Xcode.IDEFoundation.Launcher.PosixSpawn"
```
[the remark this answers is not in the document]
Comment (pmusaraj) [reply ios/Discourse.xcodeproj/xcshareddata/xcschemes/Discourse.xcscheme]: Yes, it was disabling the debugger. For some reason running the app from Xcode with Mac as the target works much better without the debugger.

**Comment by pmusaraj** [comment]
> I don't think we want to override the standard shortcuts, in any case.

It's nice to use ⌘ + W to dismiss an open site. I found a way to do that now though, by replacing the File menu.

**Review by CvX — APPROVED** [review]

**Comment by pmusaraj** [comment]
Sticking with ⌘ + W for now, but with a minor modification: if you're on the main screen, hitting ⌘ + W will dismiss the app window. Only when a site is loaded, it will dismiss that site and go back to the home screen.

**Pull request merged**